src/items/implementations.md

```diff
@@ -50,25 +50,43 @@ bracketed set of associable items.
 The nominal type is called the _implementing type_ and the associable items are
 the _associated items_ to the implementing type.
 
-Inherent implementations associate the contained items to the implementing type.
-The associated item has a path of a path to the implementing type followed by
-the associate item's path component. Inherent implementations cannot contain
-associated type aliases.
+Inherent implementations associate the contained items to the
+implementing type.  Inherent implementations can contain [associated
+functions] (including methods) and [associated constants]. They cannot
+contain associated type aliases.
+
+The [path] to an associated item is: any path to the implementing type,
+followed by the path to the associate item within the inherent
+implementation.
 
 A type can also have multiple inherent implementations. An implementing type
 must be defined within the same crate as the original type definition.
 
-```rust
-struct Point {x: i32, y: i32}
+``` rust
+pub mod color {
+    pub struct Color(pub u8, pub u8, pub u8);
 
-impl Point {
-    fn log(&self) {
-        println!("Point is at ({}, {})", self.x, self.y);
```

Comment:
Reading the whole page again, I decided to remove the old example and just use the larger example I've written since (IMHO) it more clearly demonstrates what the prose above it is discussing.